src/patternfly/components/Wizard/examples/wizard-example.hbs

```diff
@@ -0,0 +1,82 @@
+{{#> wizard id="wizard-simple-example"}}
+  {{#> background-image}}{{/background-image}}
+  {{#> wizard-header}}
+    {{#> button button--modifier="pf-m-plain pf-c-wizard__close" button--attribute='aria-label="Close"'}}
+      {{> button-icon button-icon--type="times"}}
+    {{/button}}
+    {{#> title title--modifier="pf-m-3xl pf-c-wizard__title"}}Wizard Title{{/title}}
+    {{#> wizard-description}}
+      here is where the description goes
+    {{/wizard-description}}
+  {{/wizard-header}}
+  {{#> wizard-steps}}
+    {{#> wizard-toggle}}
+      {{#> wizard-toggle-label}}
+        {{#> wizard-toggle-num}}2{{/wizard-toggle-num}}
+        {{#> wizard-toggle-step}}Configuration{{/wizard-toggle-step}}
+        {{> wizard-toggle-divider}}
+        {{#> wizard-toggle-substep}}Substep A{{/wizard-toggle-substep}}
+      {{/wizard-toggle-label}}
+      {{> wizard-toggle-icon}}
+    {{/wizard-toggle}}
+    {{#> wizard-nav}}
+      {{#> wizard-nav-list}}
+        {{#> wizard-nav-item}}
+          {{#> wizard-nav-link}}
+            Information
+          {{/wizard-nav-link}}
+        {{/wizard-nav-item}}
+        {{#> wizard-nav-item}}
+          {{#> wizard-nav-link wizard-nav-link--modifier="pf-m-current"}}
+            Configuration
+          {{/wizard-nav-link}}
+          {{#> wizard-nav-list}}
+            {{#> wizard-nav-item}}
+              {{#> wizard-nav-link}}
+                Sub-step A
+              {{/wizard-nav-link}}
+            {{/wizard-nav-item}}
+            {{#> wizard-nav-item}}
+              {{#> wizard-nav-link wizard-nav-link--modifier="pf-m-current"}}
+                Sub-step B
+              {{/wizard-nav-link}}
+            {{/wizard-nav-item}}
+            {{#> wizard-nav-item}}
+              {{#> wizard-nav-link}}
+                Sub-step C
+              {{/wizard-nav-link}}
+            {{/wizard-nav-item}}
+          {{/wizard-nav-list}}
+        {{/wizard-nav-item}}
+        {{#> wizard-nav-item}}
+          {{#> wizard-nav-link}}
+            Additional
+          {{/wizard-nav-link}}
+        {{/wizard-nav-item}}
+        {{#> wizard-nav-item}}
+          {{#> wizard-nav-link wizard-nav-link--modifier="pf-m-disabled" wizard-nav-link--attribute='aria-disabled="true" tabindex="-1"'}}
+            Review
+          {{/wizard-nav-link}}
+        {{/wizard-nav-item}}
+      {{/wizard-nav-list}}
+    {{/wizard-nav}}
+  {{/wizard-steps}}
+  {{#> wizard-main}}
+    <p>wizard step content</p>
+  {{/wizard-main}}
+  {{#> wizard-footer}}
+    {{#> form}}
+      {{#> form-actions}}
+        {{#> button button--modifier="pf-m-primary"}}
```

Comment:
I'm not sure if this is easy to do, or something we want to show here, but when I use this component in my product, I _think_ I would want this button to be `type="submit"` so that I can hit Enter in the context of the form to move on to the next page. Is that something we would want to document here, or is that something that we handle when building the React component?

Reply:
I agree. Do you think `type="submit"` should be optional, or can we just ship it that way? Seems to me like the primary action in the wizard (including if it changes from "next" to "review" or "finish") should always have `type="submit"`, so I would be fine just adding it. We do that with the Login component's form button currently. It's `type="submit"` in core and react, but not documented in either. Do you think this (and the login form button?) should be documented as needing `type="submit"`?

Reply:
I can't think of a reason why you wouldn't want `type="submit"` (or if there's a reason to use `input type="submit"` instead) so I think it would be fine to just ship it that way.

I don't think it hurts to document that, but at the same time, I don't know that we need to document it—it seems like basic, standard HTML stuff and I'm not sure where that would live in our current document structure.

Reply:
Cool! Added that in my last commit.